make floating point string formatting a MUST (#960)
ietf-wg-cellar/matroska-specification: tagging.md

#### Number Tags Formatting

`TagString` fields that require a floating-point number **MUST** use the "." mark instead of the "," mark.
Only ASCII numbers "0" to "9" and the "." character **MUST** be used.
The "." separator represents the boundary between the integer value and the decimal parts.
If the string doesn't contain the "." separator, the value is an integer value.
Thousandths separators **MUST NOT** be used.

To display it differently for another locale, applications **MUST** support auto
replacement on display.

> For legacy reasons, it is possible that the "," separator **MAY** have been used
> or that thousandths separators **MAY** have been used.
> 

Reviewer: Everything else looks perfect, but this text is an observation, not a BCP14 interoperability requirement. I'd say ```suggestion
In legacy media containers, it is possible that the "," separator or the thousandths separators might have been used.
```
Do The Right Thing, of course!

> For legacy reasons, it is possible that the "," separator **MAY** have been used
> or that thousandths separators **MAY** have been used.
> 

Reviewer: ```suggestion
In legacy media containers, it is possible that the "," separator or the thousandths separators might have been used.

```

Author: Applied. It is ineeded not a normative text.
